docs/pages/contributing/documentation.mdx

```diff
@@ -24,16 +24,14 @@ the kind of documentation we want to transform our current documentation into.
 
 </Admonition>
 
-- [How to Contribute to Teleport's
-  Documentation](./documentation/how-to-contribute.mdx): Follow this guide to
-  start contributing changes to Teleport's documentation.
-- [How to Review Documentation Changes](./documentation/reviewing-docs.mdx):
-  Read our recommended approach to preview docs changes in order to review pull
+- [How to Contribute to Teleport Documentation](./documentation/how-to-contribute.mdx) describes 
```

Comment:
Should this be "the Teleport Documentation"? Otherwise, "Teleport Documentation" sounds like it's a mass noun or missing an article.

This applies to the other instances of "Teleport documentation" in this PR as well.

Reply:
I don't think the article is necessary in this case ("how to contribute to documentation" sounds fine to me), but I can add it.